### PR TITLE
v0.7.2

### PR DESCRIPTION
**Pull request by ELundby45**
# k-means-constrained v0.7.2❄️ 

upstream: https://github.com/joshlk/k-means-constrained

## Notes
- This is a ❄️ package and will be uploaded to the ❄️ channel. The abs.yaml file will be updated upon approval
- Windows is skipped due to `or-tools` being reliant on OpenBLAS
- This is a new feedstock

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - cython 0.29.33
+    - numpy 1.23.5
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:26]: Isn't 1.23 a little high? Wouldn't it break if this package is used with a lower version that we support (for example 1.21)?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - pip
+  run:
+    - python
+    - ortools-python >=9.4
+    - scipy >=1.6.3
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:31]: Where does 1.6.3 come from? They seem ot be using >=0.13.3 upstream (https://github.com/joshlk/k-means-constrained/blob/v0.5.1/requirements.txt).

File: recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
+{% set name = "k-means-constrained" %}
+{% set version = "0.7.2" %}
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:2]:
```suggestion
{% set version = "0.5.1" %}
```

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (ELundby45) [reply recipe/meta.yaml:26]: I messed up and went with the latest tag on their GH. What we actually want is v0.7.2. This applies to all of your comments, sorry for the waste of time. 

https://github.com/joshlk/k-means-constrained/blob/master/pyproject.toml

`cython>=0.29` is specified and this is the version we have with all archs and python variants.

File: recipe/meta.yaml (hunk 2 above)
Comment (ELundby45) [reply recipe/meta.yaml:31]: https://github.com/joshlk/k-means-constrained/blob/master/requirements.txt#L2

Sorry for the 0.5.1 confusion 😞

File: recipe/meta.yaml (hunk 3 above)
Comment (ELundby45) [reply recipe/meta.yaml:2]: Sticking with 0.7.2. Thanks for pointing that out.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**